### PR TITLE
Add theblains.org servers

### PR DESCRIPTION
**Pull request by theblains**
Hi, please see the below request to add the blains.org main and testnet servers + onion.

### CODE REVIEW

**Comment by cculianu** [comment]
Hi -- we recently had a phishing attack on the network so I'm a little edgy about possible phishers infiltrating the servers.json.

Would you mind telling me a little bit about your interest in Bitcoin Cash and/or your involvement in the community.

I hate to be that guy -- since I am all about permissionlessness.  But... if you can provide me a little information, it would go a long way to assuaging my caution.

Thanks.

**Comment by cculianu** [comment]
Also your PR has some invalid JSON :) Kindly do fix that.

File: lib/servers.json [hunk 1]
```diff
@@ -96,5 +102,11 @@
         "t": "50001",
         "version": "1.4"
     }
+    "jh3jgcrwweh6yvmprtjnp72u2hqn34nlftlg3msrr4vmlapft4yvt2id.onion": {
```
Comment (cculianu) [inline lib/servers.json]: You are missing a comma here and in other places.  Json parsers will reject this file if I merge this as-is.

**Comment by markblundeberg** [comment]
Also please test out your json files and update the PR so they are valid. :D

**Comment by imaginaryusername** [comment]
Just tested cash.theblains.org:50002 and it didn't work...

**Comment by theblains** [comment]
> Hi -- we recently had a phishing attack on the network so I'm a little edgy about possible phishers infiltrating the servers.json.
> 
> Would you mind telling me a little bit about your interest in Bitcoin Cash and/or your involvement in the community.
> 
> I hate to be that guy -- since I am all about permissionlessness. But... if you can provide me a little information, it would go a long way to assuaging my caution.
> 
> Thanks.

Hi, I understand your concern. Essentially I bought  bitcoin and thus also cash a few years ago. Lately I have been reading more and seeing how I can help the community. I have recently upgraded my server which has allowed me the spare resources to run the servers and wanted to do my bit to help. Unfortunately, I cannot code but i'm comfortable running servers and wanted to help in that way. Its running ABC and Electrumx in dockers.



> Also your PR has some invalid JSON :) Kindly do fix that.

I have done my best to fix that although I am unsure why its still showing red on the Testnet page. I guess TCP ports should not be on it?



> Just tested cash.theblains.org:50002 and it didn't work...

Thats strange I have tested both inside and outside my network and it was working fine I also do have incoming connectons to electrum. Its not the perfered server setting or something? Other than that could someone else give it a try?



Thanks.

File: lib/servers_testnet.json [hunk 2]
```diff
@@ -5,11 +5,18 @@
     "blackie.c3-soft.com": {
         "s": "60002"
     },
+    "cash-testnet.theblains.org": {
+        "s" : "50004"
```
Comment (cculianu) [inline lib/servers_testnet.json]: You need commas in between items in a map (dict) in JSON.  Comma missing here is still an error...

File: lib/servers_testnet.json [hunk 3]
```diff
@@ -5,11 +5,18 @@
...
+    },
     "testnet.imaginary.cash": {
         "s": "50002"
     },
     "testnet.bitcoincash.network": {
         "s" : "60002"
+    },
+    "ebf56u6xk2e2fjlqyuz4zjj2gsrrnavfronrerh7qcvzllknytdgmmyd.onion": {
+        "s" : "50004"
+        "t" : "50003"
```
Comment (cculianu) [inline lib/servers_testnet.json]: Also need 1 comma here. See how the other entries are constructed.. :)

Also if there's only 1 item or for the last item in the list, no comma! :D

**Comment by cculianu** [comment]
Hi, well, pleased to meet you!  So long as you promise you aren't a phisher, I believe you.

FWIW I am able to connect to your server.  Do fix the JSON though.. thanks. :)

**Comment by theblains** [comment]
> Hi, well, pleased to meet you! So long as you promise you aren't a phisher, I believe you.
> 
> FWIW I am able to connect to your server. Do fix the JSON though.. thanks. :)

Pleased to meet you too! I think I have fixed it now. Thanks for your help :)

Yes I promise, plus i'm using a domain I actually use! So would not want to tarnish its name.

**Comment by cculianu** [comment]
OK, JSON fixed.  Welcome to the preferred server status! :D

You promise you're not a phiser, right? He he..

**Comment by cculianu** [comment]
Note you still had 1 last typo -- I fixed it myself. :P

**Comment by imaginaryusername** [comment]
@theblains been seeing your server down - are you still maintaining it?

**Comment by imaginaryusername** [comment]
we should probably remove his servers from servers.json before the next release.

**Comment by cculianu** [comment]
Ok, removed in latest commit: cc8c629fbd63a0f36e305ec4d3851d6ba54fe77f

**Pull request merged**